apps/indexer/src/api/repositories/drizzle.ts

```diff
@@ -128,7 +130,24 @@ export class DrizzleRepository {
       } else {
         whereClauses.push(eq(proposalsOnchain.status, status));
       }
+    } else if (status_in && status_in.length > 0) {
+      // Handle multiple statuses
+      const statusConditions = status_in.map((s) => {
+        if (
+          s === ProposalStatus.ACTIVE ||
+          s === ProposalStatus.DEFEATED ||
+          s === ProposalStatus.SUCCEEDED
+        ) {
```

Comment:
this should be handled at the service level, the DB only receives the statuses array already parsed

Reply:
Moved to service level